README.md

```diff
@@ -30,48 +30,94 @@ java -jar app-v-0.1.0.war
 ```
 
 
-<!--
+
  ## Running in Docker
 
 To run the application using Docker you have to install Docker and docker-compose on your local computer.  To install Docker see [here](https://docs.docker.com/get-docker/) and to install docker-compose see [here](https://docs.docker.com/compose/install/). 
 
 Assuming you have installed both Docker and docker-compose, to run the Onto Viewer application together with its front-end on your local machine, please make sure that you have the following folder setup:
 
 ```
-root-dir/
-    onto-viewer/                 (https://github.com/edmcouncil/onto-viewer)
-        docker/
-            runtime/
-                server/
-                    config/      <- config files go here
-                        ...
-                    ontologies/  <- ontologies go here
-                        ...
-        docker-compose.yaml
-        ...
-    html-pages/                  (https://github.com/edmcouncil/html-pages)
-        ...
++---onto-viewer
+|   |   .gitignore
+|   |   CHANGELOG.md
+|   |   CODE_OF_CONDUCT.md
+|   |   CONTRIBUTING.md
+|   |   DCO
+|   |   docker-compose.yaml
+|   |   LICENSE
+|   |   pom.xml
+|   |   README.md   
+|   +---api-doc
+|   +---docker
+|   |   |   init-onto-viewer.sh
+|   |   +---runtime
+|   |   |   \---server  
+|   |   |       +---config
+|   |   |       |       application_config.yaml
+|   |   |       |       groups_config.yaml
+|   |   |       |       label_config.yaml
+|   |   |       |       ontology_config.yaml
+|   |   |       |       search_config.yaml     
+|   |   |       |       
+|   |   |       \---ontologies
+|   |   +---server
+|   |   |       Dockerfile      
+|   |   \---web
+|   |           nuxt.config.js         
+|   +---onto-viewer-config-loader
+|   +---onto-viewer-core
+|   +---onto-viewer-toolkit
+|   +---onto-viewer-web-app
+|   \---style          
++---onto-viewer-web
```

Comment:
When I got the latest commit from this pull-request, I cannot see this folder.
Shall I create it myself? If so, how can I get the files in the folder?